1-python-question.py

```diff
@@ -25,8 +25,27 @@
 
 
 def processRequest(request):
-    # Your code here
-    return
+    try:
+        if 'applicants' not in request or len(request['applicants'])==0:
+            raise ValueError
+    except:
+        return {'error':'empty'}
+    v=5
+    m=3.5
+    banned=list(filter(lambda b:b if b in bannedVisitors else None,request['applicants']))
+    allowed=list(filter(lambda s:s if s not in bannedVisitors else None,request['applicants']))
+    output={'successfulApplicants':allowed,'bannedVisitors':banned,'TotalCost':0,'tickets':[]}
+    for i in allowed:
+        iprice=0
+        if i not in memberStatus:
+            memberStatus[i]=False
+        if memberStatus[i]==True:
+            iprice+=3.5
+        else:
+            iprice=5
```

Comment:
can just use the pricing constants that were declared earlier in lines 33 and 34